src/init.cpp

```diff
@@ -348,6 +348,10 @@ void Shutdown(NodeContext& node)
     node.chainman.reset();
     node.scheduler.reset();
 
+    if (node.fee_estimator) {
+        UnregisterValidationInterface(node.fee_estimator.get());
```

Comment:
```suggestion
        UnregisterSharedValidationInterface(node.fee_estimator);
```

if you want you can also do the check of null pointer inside the `UnregisterValidationInterface`

Reply:
Thanks, I will leave this as is since there is a check above.